src/styles/stars_style.css

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+}
+
+.stars .stars__star {
+  background-image: url(./../images/star.svg );
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(./../images/star-active.svg);
+}
```

Comment:
Due to the task description
![image](https://user-images.githubusercontent.com/80700247/229344444-a61ac877-1791-4424-be6d-3b83d10f5061.png)